### DISCUSSION TITLE
Is there a way to install notepads on windows server 2022 ? (solved)

### DISCUSSION THREAD

**Discussion by stanthewizzard** [0x7c13/Notepads#1368] [Q&A] [answered]
Hello All

Using winget I can't manageto install notepads

Thanks for help

**Comment by Estrella-Explore** [comment]
Windows server 2022 seems to be able to install `notepads` by manually installing `Microsoft store`

(I'm not sure, but I successfully installed `notepads` on Win10 LTSC 2021 using the same method)

Hope it fits you

**Comment by stanthewizzard** [comment]
cool how did you install the store ?
thanks

**Reply by Estrella-Explore** [reply]
Through [this repository](https://github.com/kkkgo/LTSC-Add-MicrosoftStore), maybe you can have a try

Although this repository has no changes since 6 years ago

lol

**Reply by Estrella-Explore** [reply]
btw i found [a nearly same name repo](https://github.com/Goojoe/LTSC-ADD-Microsoft-Store)

but i have never tried it, maybe it can help in some ways

**Reply by stanthewizzard** [reply]
THANKS !

**Reply by Estrella-Explore** [reply]
have u successfully installed Microsoft Store?

**Reply by stanthewizzard** [reply]
No LOL
Need to have python env on this server

**Reply by Estrella-Explore** [reply]
bruh

**Reply by stanthewizzard** [reply]
Tomorrow lol

**Reply by Estrella-Explore** [reply]
lol

**Comment by stanthewizzard** [comment]
installed
the store is up and running but I can't install any app on WS2022

**Reply by Estrella-Explore** [reply]
hmmm...

why.....

don't I think there should be such a problem...... 

are there any error messages related to it?

**Reply by Estrella-Explore** [reply]
I found a textbook about manually installing .appx when the Microsoft Store isn't available, but it's Chinese (I'm not a native English speaker), do you want to read it yourself or do I pack it up and send it to you?

**Reply by stanthewizzard** [reply]
I can try yes. Thanks

**Reply by Estrella-Explore** [reply]
ohhhh

I've written a simple document and tested it to work under win10 LTSC, I'll send you the link directly

https://wormhole.app/W6LZN#VhPEwMbdzFWbDgNRAR0AUg

**Reply by Estrella-Explore** [reply]
i have tested by uninstall notepads i installed before, and reinstalled it by these steps

(maybe install dependencies is not necessary, \
because i failed to install these dependencies but successfully installed notepads)

**Reply by Estrella-Explore** [reply]
@stanthewizzard

**Reply by Estrella-Explore** [reply, marked as answer]
it's now 10pm UTC+8:00, i gtg, so if there are any questions, you can leave a comment and I'll check in tomorrow

lol

**Reply by stanthewizzard** [reply]
Thanks (from France)

**Comment by Estrella-Explore** [comment]
did u make it to install notepads?

**Reply by stanthewizzard** [reply]
YES
It works !!!!

Can you explain how did you manage to have the correct files ?

**Reply by Estrella-Explore** [reply]
lol congratulations!

I looked up Notepads from [https://store.rg-adguard.net/](https://store.rg-adguard.net/) 

and used it to download the dependencies and manually install the package

**Reply by Estrella-Explore** [reply]
In short, we do what the Microsoft Store does for us 

(or maybe, I'm not sure if the Microsoft Store does anything else).

**Reply by stanthewizzard** [reply]
yes I remembered https://store.rg-adguard.net/

Huge thanks

**Comment by stanthewizzard** [comment]
thanks to you
notepads 
whatsapp (rdp)
and theme dynamique 


Are up and running

**Comment by stanthewizzard** [comment]
BAD news 
App are no more after a reboot

**Reply by stanthewizzard** [reply]
even the dependancies are gone

**Reply by Estrella-Explore** [reply]
WTF?

after reboot???

u mean files disappeared?

**Reply by stanthewizzard** [reply]
YES

**Reply by Estrella-Explore** [reply]
that's why???!!!!

can i email u in private for remote rdp connecting your server?

**Reply by Estrella-Explore** [reply]
i think the biggest possibility is that you (or your server provider) have installed a system restore program, which restores from a backup on every reboot

otherwise this is too bizarre ......

**Reply by stanthewizzard** [reply]
I'm the creator of the VM and there is no such thing :/

**Reply by Estrella-Explore** [reply]
bruh

then what is the damn why

**Reply by stanthewizzard** [reply]
No idea (this a remote desktop gateway I need to try on other WS2022)

**Reply by Estrella-Explore** [reply]
Honnêtement, mec, nous devrions vraiment essayer une autre VM pour nous assurer que ce problème bizarre peut être reproduit.

**Reply by stanthewizzard** [reply]
LOL french way. Yes I need to try

**Comment by stanthewizzard** [comment]
Just installed WS 2025
you can install apps from the store directly from webpage
nothing fancy to install

**Reply by Estrella-Explore** [reply]
Ohhh congratulations!

btw, have you found the reason for this strange thing about the disappearance of files?

**Reply by stanthewizzard** [reply]
I think that it is the temporary profil inside vmk from remote gateway
No it's on an other WS (2025) not an RDP gateway
Shall see after a reboot lol
